summary-report.py

```diff
@@ -37,16 +37,46 @@
 # User editable variables END
 ##########
 
+#To the arguments handling
+import sys
+import getopt
+
 import socket
 from pygtail import Pygtail
 import smtplib
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
 
+try: 
+  opts, args = getopt.getopt(sys.argv[1:],"hh:p:f:t:s:b:",["help="])
+except getopt.GetoptError:
+  print 'Try to use the help with the following command: summary-report.py -h'
+  sys.exit(2)
+
+for opt,arg in opts:
+  if opt == '-h':
+    print """summary-report.py has the following arguments options:
+      -p <production>
+      -f <sender email>
+      -t <recipients emails>
+      -s <smtp domain>
```

Comment:
I was referring to the help text ('smtp host' instead of 'smtp domain'). Again, just nitpicking since this change can go with the other fixes on your PR :)